DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Administrator/SearchableAdminViewModel.cs

```diff
@@ -32,5 +33,7 @@ public SearchableAdminViewModel(AdminUser adminUser)
         public bool IsLocked { get; set; }
 
         public bool IsCentreManager { get; set; }
+
+        public int? Page { get; set; }

```

Comment:
There are 3 and they're all the cards that appear in lists. I decided `Page` made more sense for them since that's what it actually was - the page of the list they appear on. I'm happy to change the name though if you think it makes more sense to be consistent with the others.

Reply:
Ah, that makes sense. I'm happy to keep these as they are. `Page` is more descriptive of the actual model itself when considering it in the context of a card.